exploratory_analysis.ipynb

```diff
@@ -1825,136 +1757,125 @@
+       "      <td>11130000.0</td>\n",
+       "      <td>603000.0</td>\n",
        "    </tr>\n",
        "  </tbody>\n",
        "</table>\n",
        "</div>"
       ],
       "text/plain": [
-       "       state           county  state_code  county_code        measure_name  \\\n",
-       "245430    IN     Boone County        18.0         11.0  Violent crime rate   \n",
-       "247672    KY     Lewis County        21.0        135.0  Violent crime rate   \n",
-       "247732    KY  Magoffin County        21.0        153.0  Violent crime rate   \n",
-       "247725    KY   Madison County        21.0        151.0  Violent crime rate   \n",
-       "247718    KY    McLean County        21.0        149.0  Violent crime rate   \n",
+       "    state                     measure_name  state_code  county_code  \\\n",
+       "477    TX  Sexually transmitted infections     35616.0     188830.0   \n",
+       "115    GA  Sexually transmitted infections      6188.0      76906.0   \n",
+       "279    MS  Sexually transmitted infections      6860.0      20077.0   \n",
+       "499    VA  Sexually transmitted infections     20349.0     106081.0   \n",
+       "301    NC  Sexually transmitted infections     11100.0      30000.0   \n",
        "\n",
-       "        measure_id   numerator   denominator   raw_value  fips_code  \\\n",
-       "245430        43.0    9.500000  40952.000000   23.197890    18011.0   \n",
-       "247672        43.0    9.000000  13877.000000   64.855516    21135.0   \n",
-       "247732        43.0    4.666667  13304.000000   35.077170    21153.0   \n",
-       "247725        43.0  168.666667  83345.666670  202.370049    21151.0   \n",
-       "247718        43.0    2.666667   9597.333333   27.785496    21149.0   \n",
-       "\n",
-       "        start_year  \n",
-       "245430      2009.0  \n",
-       "247672      2009.0  \n",
-       "247732      2009.0  \n",
-       "247725      2009.0  \n",
-       "247718      2009.0  "
+       "     measure_id  numerator  denominator  raw_value   fips_code  start_year  \n",
+       "477     33390.0   350383.0   75100757.0   237403.1  35804830.0   1491400.0  \n",
+       "115     21420.0   137386.0   29185253.0   207325.2   6264906.0    956759.0  \n",
+       "279     11025.0    65244.0    8847819.0   191435.0   6880077.0    492449.0  \n",
+       "499     17955.0    98006.0   23841006.0   157276.4  20455081.0    801988.0  \n",
+       "301     13500.0   137905.0   28414196.0   119505.5  11130000.0    603000.0  "
       ]
      },
-     "execution_count": 163,
+     "execution_count": 225,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "# top five measures with the higest raw_value in group 1\n",
+    "# Top five states with the higest raw_value in group 1\n",
     "\n",
-    "df1.sort_values(by='measure_name', ascending=False).head()"
+    "df1_state = df1.groupby(['state', 'measure_name']).sum().reset_index()\n",
```

Comment:
So this is really the top five state, measure_name combinations with highest summed raw_value. I would maybe use max instead of sum, because max won't change depending on # of rows within each group while the sum will bias towards state, measure_name combos with the most records, not just the largest raw_value. Unless, you have a reason that you prefer the sum over max?
A process note: normally you would make these corrections all in a single PR, just adding commits that fix each of the issues pointed out by the reviewer. Then when you have addressed all the comments and gotten PR approval, you would merge it in. You can still make follow up PRs with changes to the same file of course, but those would be separate PRs that address separate analyzes, a bug that wasn't discovered in the previous review process, etc. Anyway, not a big deal to me how you organize the PRs for this project, but in a bigger, more formal project this kind of organized approach to PRs is helpful (and you often have a jira ticket, etc that a PR is directly related to...)